File: java/src/main/be/steinsomers/bron_kerbosch/BronKerbosch3_gp.java
package be.steinsomers.bron_kerbosch;

public final class BronKerbosch3_gp implements BronKerboschAlgorithm {
    @Override
    public void explore(final UndirectedGraph graph, final CliqueConsumer cliqueConsumer) {
        BronKerboschDegeneracy.explore(graph, cliqueConsumer, PivotChoice.MaxDegreeLocal);
    }
}

File: kotlin/src/main/be/steinsomers/bron_kerbosch/BronKerbosch3gp.kt
package be.steinsomers.bron_kerbosch

class BronKerbosch3gp : BronKerboschAlgorithm {
    override fun explore(graph: UndirectedGraph, cliqueConsumer: CliqueConsumer) {
        BronKerboschDegeneracy.explore(graph, cliqueConsumer, PivotChoice.MaxDegreeLocal)
    }
}
